### PR TITLE
Welding shut an airlock now is logged

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
Welding shut an airlock now is logged

## Why It's Good For The Game
logging good

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
admin: welding/unwelding an airlock will be logged
code: changed a long var declaration from a single line into multiple lines for better readability
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — COMMENTED** [review]
Is this actually logged to server files though?

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1112,6 +1112,8 @@
 				welded = !welded
 				user.visible_message("[user.name] has [welded? "welded shut":"unwelded"] [src].", \
 									"<span class='notice'>You [welded ? "weld the airlock shut":"unweld the airlock"].</span>")
+				investigate_log("[key_name(user)] has [welded? "welded shut":"unwelded"] [src] which is located at [AREACOORD(src)].", INVESTIGATE_AIRLOCKS)
+				message_admins("[key_name_admin(user)] has [welded? "welded shut":"unwelded"] [src] which is located at [AREACOORD(src)].")
```
Comment (itsmeow) [inline code/game/machinery/doors/airlock.dm]: you do not need to message admins for this

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -1112,6 +1112,8 @@
 				welded = !welded
 				user.visible_message("[user.name] has [welded? "welded shut":"unwelded"] [src].", \
 									"<span class='notice'>You [welded ? "weld the airlock shut":"unweld the airlock"].</span>")
+				investigate_log("[key_name(user)] has [welded? "welded shut":"unwelded"] [src] which is located at [AREACOORD(src)].", INVESTIGATE_AIRLOCKS)
```
Comment (itsmeow) [inline code/game/machinery/doors/airlock.dm]: this should be `user.log_game`

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by EvilDragonfiend** [comment]
> Is this actually logged to server files though?

I think it's not logged, but I will ask admins to make sure

**Comment by EvilDragonfiend** [comment]
confirmed by Tyranicranger, it's not logged.
also, review addressed, and new test evidence attached

**Review by itsmeow — APPROVED** [review]

**Pull request merged**